### PR TITLE
added Proteus chain

### PR DESCRIPTION
**Pull request by Aswanth-c**
# Add New Chain <chainId>

Thanks for your pull request to add a new support in Sourcify.

If you haven't done so, please follow the instructions on [how to request chain support](https://docs.sourcify.dev/docs/chain-support/) in docs.

Please check the following items before submitting your pull request.

## Checklist

- [ ] The branch is named as `add-chain-<chainId>`.
- [ ] I haven't modified the [chains.json](../../src/chains.json) file directly.
- [ ] In [sourcify-chains.ts](../../src/sourcify-chains.ts) file
  - [ ] I've set `supported: true`.
  - [ ] I've set `monitored: false`.
  - [ ] I haven't added an `rpc` field but the one in [chains.json](../../src/chains.json) is used (if not, please explain why).
- [ ] I've added a test in [chain-tests.js](../../test/chains/chains-test.js) file.
- [ ] `test-new-chain` test in Circle CI is passing.

### CODE REVIEW

File: src/sourcify-chains.ts [hunk 1]
```diff
@@ -137,6 +137,12 @@ const sourcifyChainsExtensions: SourcifyChainsExtensionsObject = {
     monitored: false,
     contractFetchAddress: "https://triton.nautscan.com/" + TRITON_SUFFIX,
   },
+  "88002": {
+    // Shrapnel Mainnet
```
Comment (lzrscg) [inline src/sourcify-chains.ts:141]: what is this?

File: src/chains.json [hunk 2]
```diff
@@ -15108,6 +15108,24 @@
       }
     ]
   },
+  {
+    "name": "Proteus Chain",
+    "title": "Proteus Testnet",
+    "chain": "ETH",
```
Comment (lzrscg) [inline src/chains.json:15114]: why ETH?

File: src/sourcify-chains.ts (hunk 1 above)
Comment (Aswanth-c) [reply src/sourcify-chains.ts:141]: Oops I was copy-pasting the data structure and it got accidentally inserted

File: src/chains.json (hunk 2 above)
Comment (Aswanth-c) [reply src/chains.json:15114]: Because its build on top of Ethereum

**Pull request merged**